.github/workflows/build-executor.yml

```
@@ -0,0 +1,41 @@
+
+      - name: Build Docker image
+        run: |
+          export EXECUTOR_VERSION="${GITHUB_REF#refs/tags/executor_}"
+          docker build \
+            --build-arg EXECUTOR_VERSION=$EXECUTOR_VERSION \
+            -t ${{ secrets.DOCKER_REGISTRY_URL }}/executor:$EXECUTOR_VERSION \
+            -f ./src/executor/Dockerfile \
+            .
+
+      - name: Push Docker image
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:57: Double quote to prevent globbing and word splitting

(shellcheck)